Makefile

```diff
@@ -324,6 +324,7 @@ EXPOBJECTS = $(EXP_LIB_SOURCES:.cc=.o) $(LIBOBJECTS) $(TESTUTIL)
 
 TESTS = \
 	db_basic_test \
+	db_encryption_test \
```

Comment:
I think you also need to update TARGETS file. (It is a file we use internally at Facebook to build with buck).